gateway/services/profile.go

```diff
@@ -47,7 +47,7 @@ var ProfileRoutes = arbor.RouteCollection{
 		"GetProfileLeaderboard",
 		"GET",
 		"/profile/leaderboard/",
-		alice.New(middleware.AuthMiddleware([]models.Role{models.AdminRole, models.AttendeeRole, models.StaffRole, models.MentorRole}), middleware.IdentificationMiddleware).ThenFunc(GetProfileLeaderboard).ServeHTTP,
+		http.HandlerFunc(GetProfileLeaderboard).ServeHTTP,
```

Comment:
This line can also be changed to:
`
alice.New().ThenFunc(GetProfileLeaderboard).ServeHTTP,
`
but since we are not using middlewares I removed the instantiation of Alice.